tests/actions/fetchOrdersAction.test.js

```diff
@@ -0,0 +1,57 @@
+import fetchOrders from '../../app/js/actions/fetchOrder';
+
+import {
+  FETCH_ORDERS_LOADING,
+  FETCH_ORDERS_SUCCESS
+} from '../../app/js/actions/actionTypes';
+describe('fetchOrders action', () => {
+  beforeEach(() => moxios.install());
+  afterEach(() => moxios.uninstall());
+  const limit = 10;
+  const patient = '761d8414-6cb4-4456-be4f-232e6f819e06';
+
+  it(`creates FETCH_ORDERS_LOADING and FETCH_ORDERS_SUCCESS
+  action types upon success response from server when no url ispassed but with given limits`, async (done) => {
+    moxios.stubRequest(
+      `${apiBaseUrl}/order?totalCount=true&sort=desc&status=active&patient=${patient}&limit=${limit}&v=full`,
+      {
+        status: 201,
+        response: [{}],
+      },
+    );
+    const expectedTypes = [
+      FETCH_ORDERS_LOADING,
+      FETCH_ORDERS_SUCCESS,
+    ];
+    const store = mockStore({});
+    return store.dispatch(fetchOrders(null, limit, patient)).then(() => {
```

Comment:
since the `await` syntax is already available in this function, why still using `.then` to resolve `promises` in this function .

Reply:
Fixed, thanks.